src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj

```diff
@@ -109,6 +110,10 @@
       <Project>{b758fec1-35c1-4f93-a954-66dd33f6e0ec}</Project>
       <Name>Commands.RecoveryServices.Backup.ServiceClientAdapter</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Commands.RecoveryServices\Commands.RecoveryServices.csproj">
```

Comment:
Why do you need to add this reference?

Reply:
This project contains the definition of ARSVault which is being referenced by the methods in this project.